modal/sandbox.py

```diff
@@ -490,7 +494,7 @@ async def exec(
         _pty_info: Optional[api_pb2.PTYInfo] = None,
     ):
         """Execute a command in the Sandbox and return
-        a [`ContainerProcess`](/docs/reference/modal.ContainerProcess#modalcontainer_process) handle.
+        a [`ContainerProcess`](https://modal.com/docs/reference/modal.ContainerProcess#modalcontainer_process) handle.
```

Comment:
I think we are intentionally not using fully qualified domains in docs links, so that we can publish staging docs etc. on other domains for testing purposes and have links refer to that domain.